samples/business-process-samples/product/process-tools/src/main/java/org/wso2/ei/businessprocess/utils/processcleanup/CleanupExecutor.java

```diff
@@ -211,10 +211,10 @@ private static boolean deletePackages(String packageName) throws Exception {
 					cleanProcessInstances(processList, conn);
 				}
 				for (String id : processList) {
-					conn.createStatement().execute(query.deleteFromOdeProcess(id));
+                    executeStatement(conn, query.deleteFromOdeProcess(id));
```

Comment:
Shall we fix the formatting?

Reply:
old lines have tab formating which is incorrect